### PR TITLE
Add the UCL ARC JD framework

### PR DESCRIPTION
**Pull request by jamespjh**

### CODE REVIEW

**Comment by jamespjh** [comment]
@jonc125 added the ARC JD framework to the repo - please review and merge.

File: ucl-arc/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+For staff convenience, individual JDs composed of the appropriate sections from this document may be used as summary
+documents for specific staff or recruitments. 
+
+Each such sub-JD will consist of the preamble, text defining the seniorities appropriate to the grade, 
+and text defining the expertise associated with a job family and specialisation, taken from this document. 
+
+However, the generic JD framework here is the normative document in force.
+
+For each of the  job families, there are a number of possible equivalent titles (separated by a slash “/” below) – staff are able to choose one of these according to preference and roles will be advertised with the option. (To facilitate staff self-identifying in roles that match their career aspirations.)
+Seniority Prefixes: Junior (G6), no-prefix (G7), Senior (G8), Principal (G9) and Professorial (G10)
```
Comment (jonc125) [inline ucl-arc/readme.md:16]: We're not using Junior any more! I've updated the wiki source.
(Similarly everywhere in this PR.)

**Review by jonc125 — CHANGES_REQUESTED** [review]
The repo is now organised by grade so ucl appears in 3 places currently which we should remove and replace with this content. That's a bit awkward for us! I'd suggest we put this in the 'standard' role and link to it from the others.

We should also include the draft promotions process text.

File: ucl-arc/readme.md (hunk 1 above)
Comment (jamespjh) [reply ucl-arc/readme.md:16]: I tried to click to merge your changes removing junior, but think I merged the master PR instead. D'oh.

**Pull request merged**